test/integration/scheduler_perf/scheduler_perf_test.go

```diff
@@ -637,39 +643,33 @@ func runWorkload(b *testing.B, tc *testCase, w *workload) []DataItem {
 				b.Fatalf("op %d: %v", opIndex, err)
 			}
 			b.Cleanup(func() {
-				nsPreparer.cleanup()
+				if err := nsPreparer.cleanup(); err != nil {
+					b.Fatalf("failed to cleanup namespace: %v", err)
+				}
 			})
 
 		case *createPodsOp:
-			var namespace string
+			namespace := defaultNamespace
 			if concreteOp.Namespace != nil {
 				namespace = *concreteOp.Namespace
-			} else {
-				// define Pod's namespace automatically, and create that namespace.
-				namespace = fmt.Sprintf("namespace-%d", opIndex)
-				_, err := client.CoreV1().Namespaces().Create(ctx, &v1.Namespace{ObjectMeta: metav1.ObjectMeta{Name: namespace}}, metav1.CreateOptions{})
-				if err != nil && !apierrors.IsAlreadyExists(err) {
-					b.Fatalf("failed to create namespace for Pod: %v", namespace)
-				}
-				b.Cleanup(func() {
-					err := client.CoreV1().Namespaces().Delete(ctx, namespace, metav1.DeleteOptions{})
-					if err != nil {
-						b.Errorf("failed to delete namespace %v", namespace)
-					}
-				})
 			}
+			// indexStr is used for label.
+			// All pods which are created in this op will have this index in label,
+			// and we will use that label in labelselector to wait all Pods to be scheduled.
+			indexStr := strconv.Itoa(opIndex)
```

Comment:
Didn't we decide to just wait for ALL pods in the namespace to be scheduled?

I'm just looking for simplicity here.